.devcontainer/host_setup.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
```

Comment:
This whole script is basically a duplicate of what's published here: https://osquery.readthedocs.io/en/stable/development/building/#linux-ubuntu-18

Reply:
This makes a third place these live.
1. Docs
2. CI pipeline
3. here

Is there a way we can unify these?

Reply:
What do we think of this possible change to the docs?
https://github.com/NickBorgers/osquery/blob/30c2cccc65b44cd85af8d70573f40f14e5b50df3/docs/wiki/development/building.md#linux-ubuntu-18